src/main/java/com/sprint/mission/discodeit/entity/Channel.java

```diff
@@ -0,0 +1,42 @@
+package com.sprint.mission.discodeit.entity;
+import java.util.ArrayList;
```

Comment:
패키지와 import 는 한줄 띄는게 좋습니다.  

```suggestion
package com.sprint.mission.discodeit.entity;

import java.util.ArrayList;
```